app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -48,11 +48,11 @@ If you are already a qualified teacher, you do not need to retrain to teach in E
 * to apply for English 'qualified teacher status', or be able to demonstrate strong non-UK qualifications and experience 
 * a high standard of written and spoken English
 * to pass criminal and professional safeguarding checks (these will be organised by your employer)
-* a visa or immigration status allowing you to work in the UK
+* [a visa or immigration status allowing you to work in the UK](#apply-for-your-visa)
```

Comment:
```suggestion
* [a visa or immigration status allowing you to work in the UK](/non-uk-teachers/visas-for-non-uk-teachers)
```
I don't think there is much point jumping to an anchor link on this page which then just redirects the user to the separate visas page - might as well go straight to the visas page?

cc @Sarah-DfE - if you agree, please commit this!